ResearchKit/ActiveTasks/ORKAccelerometerRecorder.m

```diff
@@ -192,18 +190,27 @@ @implementation ORKAccelerometerRecorderConfiguration
 
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Wobjc-designated-initializers"
-- (instancetype)initWithFrequency:(double)freq {
-    self = [self ork_init];
+
+- (instancetype)initWithIdentifier:(NSString *)identifier {
```

Comment:
If it's really invalid to call this, this initializer should be marked NS_UNAVAILABLE in the header.